bakerydemo/templates/includes/card/listing-card.html

```diff
@@ -1,12 +1,28 @@
 {% load wagtailimages_tags %}
 
 <li class="listing-card">
-    <a class="listing-card__link" href="{{ childpage.url }}">
-        <figure class="listing-card__image">
-            {% image childpage.image fill-180x180-c100 %}
-        </figure>
+    <a class="listing-card__link" href="{{ page.url }}">
+        {% if page.image %}
+            <figure class="listing-card__image">
+                {% image page.image fill-180x180-c100 %}
+            </figure>
+        {% endif %}
         <div class="listing-card__contents">
-            <h3 class="listing-card__title">{{ childpage.title }}</h3>
+            <h3 class="listing-card__title">{{ page.title }}</h3>
+            <table class="listing-card__meta">
+                {% if page.origin %}
+                    <tr>
+                        <td class="listing-card__meta-category">Origin</td>
+                        <td class="listing-card__meta-content">{{ page.origin }}</td>
+                    </tr>
+                {% endif %}
+                {% if page.bread_type %}
+                    <tr>
+                        <td class="listing-card__meta-category">Type</td>
+                        <td class="listing-card__meta-content">{{ page.bread_type }}</td>
```

Comment:
I combined the css rules that apply to these  classes in an earlier comment but I wonder if it's worth having 1 class for both instead?

Reply:
The combination is the right idea for the desktop styles in the media query, however for mobile styling they have slightly different colours `--grey` and `--dark` so I think the class names should be kept seperate

Reply:
Ah my bad, I didn't stop the styles were different on mobile. Definitely worth keeping separate : )